dart/tool/dart/generate_dart.dart

```diff
@@ -299,6 +358,8 @@ class Api extends Member with ApiParseUtil {
     } else {
       throw 'unexpected entity: ${name}, ${definition}';
     }
+    _mergeTypes();
```

Comment:
Could you add a line doc here describing that we're merging the field for types and enums from the service.md and service_undocumented.md files?